static/css/directives/ui/repo-list-table.css

```diff
@@ -19,6 +19,11 @@
   font-size: 13px;
 }
 
+.repo-list-table .state {
```

Comment:
Use `repo-state` in case there is a name conflict with another CSS class. Just `state` is pretty common.